mkdocs.yml

```diff
@@ -37,3 +37,4 @@ plugins:
 nav:
     - Home: index.md
     - Credentials: credentials.md
+    - Jobs: jobs.md
```

Comment:
I think we need to add service, exceptions here.

Reply:
I'll add exceptions, but not service because that will be included later when the corresponding tasks exist